setup.cfg

```diff
@@ -9,7 +9,7 @@ python_files = *Test.py
 python_classes = *Test
 python_functions = *_test
 timeout = 35
-addopts = --color=yes
+addopts = --color=yes --ignore=test-requirements.txt
```

Comment:
`norecursedirs` only works for directories and not a single file. So, even I think this is the only possible solution